kubernetes/mariadb/skip-networking.cnf

```diff
@@ -1 +1,2 @@
-skip-networking
\ No newline at end of file
+[mysqld]
+skip-networking
```

Comment:
Sorry, this was meant to be for the whole commit, not only this bit. Take into account that very soon this manifests are going to be removed and we will only use the production ones (with some automatic tweaking): https://github.com/kubic-project/velum/pull/135